src/rt/test/systematic.h

```diff
@@ -362,6 +364,7 @@ namespace Systematic
     }
 #endif
 
+#ifdef USE_FLIGHT_RECORDER
```

Comment:
If you push the `ifdef` inside the body, you will have a smaller diff.

Reply:
Changed it to "if constexpr" and moved it around the output to keep the diff smaller.